vendor.conf

```diff
@@ -1,57 +1,57 @@
 github.com/Shopify/logrus-bugsnag   6dbc35f2c30d1e37549f9673dd07912452ab28a5
-github.com/sirupsen/logrus          f006c2ac4710855cf0f916dd6b77acf6b048dc6e  # v1.0.3
+github.com/sirupsen/logrus          8bdbc7bcc01dcbb8ec23dc8a28e332258d25251f # v1.4.1
 github.com/bugsnag/bugsnag-go       13fd6b8acda029830ef9904df6b63be0a83369d0
-github.com/bugsnag/panicwrap	    e2c28503fcd0675329da73bf48b33404db873782
-github.com/bugsnag/osext	    0dd3f918b21bec95ace9dc86c7e70266cfc5c702
+github.com/bugsnag/panicwrap        e2c28503fcd0675329da73bf48b33404db873782
```

Comment:
These now do releases as well; this one is one commit ahead of v1.0.0; https://github.com/bugsnag/panicwrap/compare/1.0.0...e2c28503fcd0675329da73bf48b33404db873782 - one behind v1.1.0; https://github.com/bugsnag/panicwrap/compare/e2c28503fcd0675329da73bf48b33404db873782...1.1.0

v1.2.0 now un-forked  github.com/bugsnag/osext, I see https://github.com/bugsnag/panicwrap/compare/e2c28503fcd0675329da73bf48b33404db873782...v1.2.0